src/testing/task_plot_share_of_hh_members_demanding_test.py

```diff
@@ -21,42 +20,33 @@
     }
 )
 @pytask.mark.produces(
-    BLD / "data" / "testing" / "share_of_hh_members_with_rapid_test_after_hh_event.png"
+    BLD / "figures" / "data" / "testing" / "private_test_demand_shares.png"
 )
-def task_plot_share_of_workers_receiving_test_offer(depends_on, produces):
-    rki = pd.read_pickle(BLD / "data" / "processed_time_series" / "rki.pkl")
-    rki = rki.groupby("date")["newly_infected"].sum()
-    rki = rki.rolling(7).sum().dropna()
-    rki = 100_000 * rki / POPULATION_GERMANY
-    rki = rki["2021-03-01":]
-
+def task_plot_private_test_demand_shares(depends_on, produces):
     params = pd.read_pickle(depends_on["params"])
     with warnings.catch_warnings():
         warnings.filterwarnings(
             "ignore", message="indexing past lexsort depth may impact performance."
         )
         params_slice = params.loc[("rapid_test_demand", "private_demand")]
-    share_hh_members_demanding_test = get_piecewise_linear_interpolation(params_slice)
-    share_hh_members_demanding_test = share_hh_members_demanding_test.loc[
-        "2021-03-01" : rki.index.max()
-    ]
 
-    fig, axes = plt.subplots(2, figsize=(10, 10), sharex=True)
-    sns.lineplot(x=rki.index, y=rki, ax=axes[0])
-    axes[0].grid()
-    axes[0].set_title("Incidence")
+    private_demand_shares = get_piecewise_linear_interpolation(params_slice)
+    private_demand_shares = private_demand_shares.loc["2021-01-01":"2021-07-01"]
 
+    fig, ax = plt.subplots(figsize=(8, 6))
     sns.lineplot(
-        x=share_hh_members_demanding_test.index,
-        y=share_hh_members_demanding_test,
-        ax=axes[1],
+        x=private_demand_shares.index,
+        y=private_demand_shares,
+        ax=ax,
     )
-    axes[1].grid()
-    axes[1].set_title(
-        "Share of Household Members Demanding Rapid Test\n"
-        "When Household Member Tests Positive Or Becomes Symptomatic"
+    ax.set_title(
```

Comment:
This could also be solved in Latex if it used in the publication. I think it would look better.

Thinking about this, we could align the format of the text and figures for publication more by using serif fonts in the figures. Shameless self-plug https://tobiasraabe.github.io/blog/matplotlib-for-publications.html#matplotlib-for-publications

Reply:
I think at some point we will have all the titles only in LaTeX. At the moment there is some doubling so the figures provide enough context outside LaTeX. However, for legends etc it would probably be good to use the same font as in the paper. Let's do this in a future PR.